### PR TITLE
Unit conversion docs

### PR DESCRIPTION
**Pull request by javerbukh**
Documentation for unit conversion.

### CODE REVIEW

**Comment by javerbukh** [comment]
@brechmos-stsci @eteq I have written some documentation for unit conversion, however this is my first time writing a .rst file, so I'm not sure if this is how it should look. Can you guys let me know about any changes that need to be made?

File: docs/unit_conversion.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Unit Conversion
+===============
+
+How Unit Conversion Works
+-------------------------
+
+Unit Conversion uses the `astropy.units <http://docs.astropy.org/en/stable/units/>`_ module in order to convert spectral
+axis and flux units.
+
+    import astropy.units as u
```
Comment (nmearl) [inline docs/unit_conversion.rst]: This should be placed in a code block:

```
.. code-block:: python

    import astropy.units as u
    potential_X_axis_units = u.Unit(original_spectral_axis_units).find_equivalent_units(equivalencies=u.spectral())

    ...
```

File: docs/unit_conversion.rst [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+Unit Conversion
+===============
+
+How Unit Conversion Works
+-------------------------
+
+Unit Conversion uses the `astropy.units <http://docs.astropy.org/en/stable/units/>`_ module in order to convert spectral
+axis and flux units.
+.. code-block:: python
```
Comment (eteq) [inline docs/unit_conversion.rst]: I think maybe you need a space above this directive?  Usually rst depends on blank lines to recognize things like this...

**Comment by nluetzge** [comment]
How about adding a screenshot to to the main description? Just something like that:
screen shot 2018-11-08 at 1 34 46 pm

Otherwise it looks good to me.

**Pull request merged**